### ISSUE TITLE
Min, Max and close button cannot be seen

### ISSUE THREAD

**Issue by engkhsky** [downthemall/downthemall-legacy#246]
### Your DownThemAll! and Browser Version and Language and Operating System
DownThemAll: 3.0.8
Firefox: 55.0.2
OS: Win 7

### Actual Behavior
![image](https://user-images.githubusercontent.com/24392183/29393084-5c32dd70-8334-11e7-9cba-f5c3e6b6f99f.png)

The min, max and close button are missing.

### Additional info such as log extracts
*Please enable Diagnostic Logging in the Privacy tab of the DownThemAll! preferences and either copy/paste essential part of the log or attach the entire log. Please note that the log may include sentitive information which you have to remove prior to posting it!*

**Comment by hook321** [comment]
I think the close button not showing up is also reported in https://github.com/downthemall/downthemall/issues/242

**Comment by econrad003** [comment]
This problem has something to do with changes in Firefox 55 - It occurs with 55.0.1 as well.  It may also be related to the theme in use -- I was using Compact Dark.  I dropped back to Firefox 54.0.1 and the top of the window was restored, but I had to right click there to get the tools (_e.g._ close).  When I switched from Compact Dark to the default theme, the close icon was restored as well.  [Before the update, I'm pretty sure that the close icon was in the DTA manager under Compact Dark.]

I haven't tried using DTA with the default theme in Firefox 55...  Hope that helps!

**Comment by hook321** [comment]
Once v57 ESR comes out you guys should really either update and find an alternative addon that is compatible, or switch to a Pale Moon if you wish to continue using DTA. Continuing to use an old version of Firefox will become more and more of a security risk as time goes on.

**Comment by ianamk** [comment]
palemoon is not a real solution it has one developer that forks mozilla code when mozilla goes electrolysis so will palemoon and addons will not work there either
I read flashgot was going to get a webext update, I'm hoping the mozilla people hear extension developers and allow for low level access and dta continues to live if not flashgot + an external download manager will have to do.

**Comment by ianamk** [comment]
this seams to be a win7 only issue the theme used does not matter the window border is gone with the default the new compact or a 3rd party theme from mozilla themes.
I even tried disabling hardware acceleration it made no difference, I even thought it might be aero but this problem occurs with aero disabled ie windows classic theme

**Comment by reign70** [comment]
Same here with the new firefox update 55.0.2 and window 7

**Comment by hook321** [comment]
@ianamk I've been using Pale Moon for a long time now, and it works fine for the most part, and they seem to care about the community more than Mozilla does. Working around the results of Mozilla's poor decisions is not a real solution. Extension developers are free to continue developing their extension for Pale Moon if they want to, but they can of course choose not to.

**Comment by ianamk** [comment]
@hook54321a I've used PaleMoon in the past it's not bad browser but it's based on 3 years old Firefox code it has only one full time developer it's not a real choice if you want to use classic Firefox addons you're be better off using Firefox-esr
but this bug has nothing to do with PaleMoon or e10 it only affects windows 7 users which is rather strange and probobly won't get a fix

**Comment by John13579** [comment]
I'd also like to point out I’m having exact same trouble as the TC has.
The min/max/close buttons on the new version (v3.0.8) also disappeared for me as well, mine looks exactly like what TC has, and I have no themes installed at all.

I also can't move the DTA window around at all.
Firefox: v55.0.2
OS: W7:64-bit

PLEASE fix this bug/glitch, or at least give is us a work-around that works for now, I just love the add-on!

**Comment by ianamk** [comment]
I do believe this was fixed in the current trunk anybody having this issue should try the current nightly trunk build
https://code.downthemall.net/nightly/downthemall-nightly-trunk.xpi
hopefully the devs will push an update on the mozilla addons site

**Comment by reign70** [comment]
In fact the buttons are there if you put the mouse cursor over it and they are still working but they are not visible

![2017-08-20_17-02-05](https://user-images.githubusercontent.com/31149194/29498839-86b14040-85c9-11e7-9475-16f552081c3f.jpg)

**Comment by ianamk** [comment]
it's a bug that is fixed in the current nightly trunk
![screen-2017-08-21_01-20-13](https://user-images.githubusercontent.com/28848990/29499347-413391ee-860f-11e7-80e2-8702312ba553.png)
hopefully the fix will make it's way to MOA

**Comment by reign70** [comment]
I've read ianamk about nightly version before but for me is not working, it has some other display issues, and also firefox keeps running even when i close it. nightly versions are not a a good idea if you are not developing as mentioned here:  https://code.downthemall.net/nightly/

![display](https://user-images.githubusercontent.com/31149194/29499674-7ddd9102-8606-11e7-9b6e-ba9006226a6b.jpg)
![2017-08-20_19-16-41](https://user-images.githubusercontent.com/31149194/29499675-7dff91b2-8606-11e7-9392-a41f8f667cc4.jpg)
![2017-08-20_19-17-36](https://user-images.githubusercontent.com/31149194/29499676-7dffff8a-8606-11e7-8413-7e8cdcb71ffc.jpg)
![2017-08-20_19-18-07](https://user-images.githubusercontent.com/31149194/29499677-7e0722ce-8606-11e7-81a5-87ade923eb2e.jpg)

**Comment by ianamk** [comment]
@reign70 
the nightly has a different GUID so it gets installed in parallel but it uses the same preference as the release build you basicly have 2 different addons accessing the same preferences and bugs are sure to follow, you should disable (not remove) the release build and install the nightly-trunk not the nightly-dta-3.0 as that one doesn't work
I've used nightly-trunk for a few days and had no issues with firefox not closing or anything else and I've got 20+ addons installed
https://imgur.com/a/WHd4x
leaving the release build disabled will still pull updates and when version 3.09 is out you can disable the nightly and continue using the release build

**Comment by Nono-m0le** [comment]
Still having the same issue here. (win7 as well)

**Comment by southern-boy** [comment]
I downloaded the nightly and because it was in parallel with the regular 3.0.8 neither worked. I had to uninstall.  I continue to have the missing close box.  It is there when first accessed but when I select to download some files it is missing.  Someone earlier suggested that the close box was there but hidden.  It is not hidden in mine.  If I mouse over the upper right corner the link shown is the donate link.

Downthemall is a great product.

**Comment by reign70** [comment]
yes, the nightly version does not work, the close button is there, as well as maximize and close button, but there are hidden, only when you mouse over in that zone and they are very thin (see attached)

![2017-10-16_19-01-34](https://user-images.githubusercontent.com/31149194/31640532-b66871a2-b2a4-11e7-9a51-1b9bd5650ab1.jpg)

**Comment by reign70** [comment]
Minimize, maximize and close buttons

![2017-10-16_19-04-55](https://user-images.githubusercontent.com/31149194/31640592-f9ae8384-b2a4-11e7-84d1-b6f96a67e941.jpg)
![2017-10-16_19-04-36](https://user-images.githubusercontent.com/31149194/31640589-f8fadb2c-b2a4-11e7-99e5-81eef0b18106.jpg)
![2017-10-16_19-04-45](https://user-images.githubusercontent.com/31149194/31640590-f934a866-b2a4-11e7-85f5-16cc153c467c.jpg)

**Comment by ianamk** [comment]
works for me
![clipboard01](https://user-images.githubusercontent.com/28848990/31647471-b50fd138-b308-11e7-9896-2a717be58f37.png)
there are 2 nightly versions make sure you've got the right one get the trunk one and make sure the stable version is disabled or removed
this one works great for me
https://code.downthemall.net/nightly/downthemall-nightly-trunk.xpi
sadly in a few weeks downthemal!l will be dead :(
